indico/http/client.py

```diff
@@ -15,45 +14,47 @@
 
 logger = logging.getLogger(__file__)
 
+
 class CookiePolicyOverride(http.cookiejar.DefaultCookiePolicy):
     def set_ok(self, cookie, request):
         return True
+
     def return_ok(self, cookie, request):
         return True
+
     def path_return_ok(self, path, request):
         return True
+
     def domain_return_ok(self, domain, request):
         return True
 
+
 class HTTPClient:
-    def __init__(self, config: IndicoConfig=None):
+    def __init__(self, config: IndicoConfig = None):
```

Comment:
Kind of weird it needs spaces when there is a type for not when it's just a kwarg.

Reply:
black formatter does the thinking so we don't have to debate it :shrug:  .... except line width